components/omega/doc/design/Decomp.md

```diff
@@ -0,0 +1,260 @@
+<!--- Omega Decomposition Requirements and Design ----------------------------->
+
+# OMEGA Requirements and Design:
+
+## *Decomp*
+
+
+## 1 Overview
+
+For parallel execution, OMEGA will utilize a hierarchical parallelism with the
+coarse-grained parallelism achieved via message passing. An OMEGA mesh will be
+decomposed into subdomains so that computations on each sub domain can proceed
+in parallel and messages will be passed to communicate data between domains. 
+This document describes the requirements and design of this domain 
+decomposition.
+
+
+## 2 Requirements
+
+### 2.1 Requirement: Mesh partition
+
+The primary requirement is to be able to partition a mesh across a
+distributed memory machine. Many standard tools exist to perform
+this partitioning (eg metis, parmetis, zoltan) and their use is
+encouraged. This normally requires an input file with the primary
+mesh indices and adjacency (see Requirement 2.4).
```

Comment:
The requirements don't explicitly say that the partition tools will re-index all the index variables from the global index to the local index (`cellsOnCell`, `edgesOnCell`, etc). I think it is implied here, but should be stated. Specifically, that only the global indices are stored for these variables in the file, and that the re-indexing is done  on start-up so that the local variables have the local versions of these indices.

Reply:
This is described in 2.7. I have clarified that section a little.